cli/src/opts/cast.rs

```diff
@@ -298,6 +301,17 @@ pub enum Subcommands {
         #[structopt(subcommand)]
         command: WalletSubcommands,
     },
+    #[structopt(name = "interface", about = "generate contract's interface from ABI")]
+    Interface {
+        #[structopt(help = "The contract's address")]
+        contract_address: Option<String>,
```

Comment:
I think it should be possible to do `Option<Address>` here, given that `Address` implements `FromStr`.